mautic/mautic
# Re-enable rendering of html in form field labels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #1790 


## Description

The ability to use html in form field labels (e.g. for links) was broken in Mautic 5. This PR re-enables it.


---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a form with multiple types of fields (Checkbox, Email, ...)
3. For each of those fields, use a label that contains html, for example `<a href="https://mautic.org"> Click here </a>`
4. See that the html is rendered
<img width="804" alt="Bildschirmfoto 2024-07-30 um 14 53 35" src="https://github.com/user-attachments/assets/cd35bc58-6b67-4e52-95ff-b53baf41cdcb">

6. Save the form
7. Verify that the html is also rendered in /s/forms/<id> and in the preview
![image](https://github.com/user-attachments/assets/3f3e2497-7b3a-4eb0-99f8-d5bc42ab8d17)
